frontend/src/components/Tree/Tree.jsx

```diff
@@ -92,7 +102,22 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick) {
     .attr("text-anchor", "middle")
     .attr("font-size", 18)
```

Comment:
Maybe we should reduce font size a bit? The text feels too clustered sometimes.

Reply:
I'm dropping it to 13 due to the tree for PSY359 having a lot of nodes; please look over and tell me what you think

Reply:
Think it looks good!